test/async-hooks/test-graph.tcp.js

```diff
@@ -17,7 +17,7 @@ const server = net
 
 server.listen(common.PORT);
 
-net.connect({ port: server.address().port, host: server.address().address },
+net.connect({ port: server.address().port, host: '::1' },
```

Comment:
But it will resolve to `127.0.0.1`.
I'm trying to figure out if `::` was just used for convenience, or if IPv6 has any significance.